src/components/List.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import useFirestore from '../hooks/useFirestore';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+  const history = useHistory();
+
   return (
     <div>
       <h1>List</h1>
+      {docs.length === 0 && (
+        <div>
+          <p>Your shopping list is currently empty.</p>
+          <button onClick={() => history.push('/add-item')}>Add an Item</button>
+        </div>
+      )}
       <ul>{docs && docs.map((doc) => <li key={doc.id}>{doc.itemName}</li>)}</ul>
     </div>
   );
 };
 
 export default List;
+
+//Issue 7 AC:
+// The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
```

Comment:
Should we remove these comments?

Reply:
Yes, we should def remove those. And I like the idea of a loader. That would be a nice solution to that little bug.

Reply:
+1 to the idea of a loader! 😄  The flash of "you have no items in the list" while the data is loading could be confusing or jarring for our hypothetical users.